Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -474,6 +475,17 @@ private static void AddTimeSeekResponseHeaders(StreamState state, IHeaderDiction
                 }
             }
 
+            // Fallback to the container of mediaSource
+            if (!string.IsNullOrEmpty(mediaSource?.Container))
+            {
+                var containers = mediaSource.Container.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+
+                if (containers.Length > 0)
+                {
+                    return '.' + containers[0];
+                }
```

Comment:
I think this is identical
```suggestion
                var idx = mediaSource.Container.IndexOf(',');
                return '.' + (idx == -1 ? mediaSource.Container : mediaSource.Container[..idx]);
```

Reply:
It works.